src/maks.kulykov/Main.java

```diff
@@ -0,0 +1,52 @@
+package maks.kulykov;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        System.out.println("Hello!");
+        System.out.println("Enter the name of the database you want to work with:");
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        String dbName = reader.readLine();
+        System.out.println("Enter your user name:");
+        String userName = reader.readLine();
+        System.out.println("Enter your password:");
+        String password = reader.readLine();
+
+        DatabaseManager manager = new DatabaseManager(dbName, userName, password);
+
+        boolean isConnection = false;
+
+        while (!isConnection) {
+            switch (manager.checkCredentials()) {
+                case "db" -> {
+                    System.out.println("There is no database with this name. Please enter correct database:");
+                    dbName = reader.readLine();
+                    manager = new DatabaseManager(dbName, userName, password);
```

Comment:
it is also weird to create a DatabaseManager again. So need to rewrite this logic with passing credentials as parameters to 'checkCredentials()'.
Also think about to call method not in every case but once after switch().

Reply:
+